games-emulation/higan/higan-116_pre.ebuild

```diff
@@ -0,0 +1,100 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit desktop toolchain-funcs xdg
+
+DESCRIPTION="A multi-system emulator focused on accuracy, preservation, and configurability"
+HOMEPAGE="https://github.com/higan-emu/higan"
+SRC_URI="https://github.com/higan-emu/${PN}/archive/nightly.tar.gz -> ${P}.tar.gz"
```

Comment:
Right. This doesn't work. They don't have a recent versioned tarball available?'

Reply:
If you want to use what they call `higan nightly 2021-03-23`, since they do not make it available as a reliable tarball, you'll have to make a snapshot pinned to the specific commit 44cbb88d49d5a77f19e353fad4bc7fbab6909ca9 and the ebuild will then be versioned `116_pre20210323`.

Reply:
Ah, I see what's going on: this nightly version isn't static but is periodically updated. I talked with the team last December and they hadn't chosen a date at the time for the 116 proper release; if I recall correctly, the intention was to release at some time this year.

Pinning this ebuild to a snapshop might be the best path forward for now. Does Github provide a way to get the tarball of a specific commit rather than a tag?

Reply:
Looks like it's as simple as replacing `nightly` with the commit ID, so I'll do that force-push the update.